### DISCUSSION TITLE
New UI Broke Classroom Options Dropdown Positioning in Firefox and Safari but not Chrome

### DISCUSSION THREAD

**Discussion by chrissphinx** [community/community#60453] [GitHub Education]
### Hi everyone,

C'mon folks, are we really only testing UI changes with a single browser??

**Comment by chrissphinx** [comment]
Can we please get an issue tracker for GitHub Classroom that is publicly accessible so we can properly report things like this? Having to do it on the GitHub Education forums is embarrassing

**Comment by chrissphinx** [comment]
Actually, it appears now on **all three** browsers that the editing assignment prefix:
<img width="614" alt="CleanShot 2023-07-09 at 14 31 12@2x" src="https://github.com/community/community/assets/2583639/2d727322-5eac-42f2-8c7e-3dca5609e16a">
just causes it to disappear:
![CleanShot 2023-07-09 at 14 33 16](https://github.com/community/community/assets/2583639/dcdf1cac-e01c-4771-aa01-71dff7589738)
excuse gif quality, but hopefully the issue is clear

would love to open a PR to fix this! :)

**Comment by RyzenFromFire** [comment]
Can confirm this is still a problem. Oddly enough it doesn't work in Brave, which is Chromium based.

**Reply by RyzenFromFire** [reply]
Just tested in Chrome as well, and it doesn't work - previously didn't see OP's note that it doesn't work in any browser now.

**Comment by HugoHenriql** [comment]
Pode confirmar que isso ainda é um problema. Curiosamente, não funciona no Brave, que é baseado no Chromium.

**Comment by chaserhkj** [comment]
Can confirm this still happens, even in Chrome as of now.
